doc/source/cluster/quickstart.rst

```diff
@@ -0,0 +1,240 @@
+            cluster_name: minimal
+            
+            # Cloud-provider specific configuration.
+            provider:
+                type: azure
+                location: westus2
+                resource_group: ray-cluster
+            
+            # How Ray will authenticate with newly launched nodes.
+            auth:
+                ssh_user: ubuntu
+                # you must specify paths to matching private and public key pair files
+                # use `ssh-keygen -t rsa -b 4096` to generate a new ssh key pair
+                ssh_private_key: ~/.ssh/id_rsa
+                # changes to this should match what is specified in file_mounts
+                ssh_public_key: ~/.ssh/id_rsa.pub
+
+    .. group-tab:: GCP
+
+        .. code-block:: yaml
+
+            # A unique identifier for the head node and workers of this cluster.
+            cluster_name: minimal
+            
+            # Cloud-provider specific configuration.
+            provider:
+                type: gcp
+                region: us-west1
+
+Save this configuration file as ``config.yaml``. You can specify a lot more details in the configuration file: instance types to use, minimum and maximum number of workers to start, autoscaling strategy, files to sync, and more. For a full reference on the available configuration properties, please refer to the :ref:`cluster yaml configuration options reference <cluster-config>`.
+
+After defining our configuration, we will use the Ray Cluster Launcher to start a cluster on the cloud, creating a designated "head node" and worker nodes. To start the Ray cluster, we will use the Ray CLI. Run the following command:
+
+.. code-block:: shell
+
+    $ ray up -y config.yaml
+
+Run the application in the cloud
+--------------------------------
+
+We are now ready to execute the application in across multiple machines on our Ray cloud cluster. Run the following command:
+
+.. code-block:: shell
+
+    $ ray submit config.yaml script.py
+
+The output should now look similar to the following:
+
+.. parsed-literal::
+
+    This cluster consists of
+        3 nodes in total
+        6.0 CPU resources in total
+    
+    Tasks executed
+        3425 tasks on xxx.xxx.xxx.xxx
+        3834 tasks on xxx.xxx.xxx.xxx
+        2741 tasks on xxx.xxx.xxx.xxx
+
+In this sample output, 3 nodes were started. If the output only shows 1 node, you may want to increase the ``secs`` in ``time.sleep(secs)`` to give Ray more time to start additional nodes.
+
+The Ray CLI offers additional functionality. For example, you can monitor the Ray cluster status with ``ray monitor config.yaml``, and you can connect to the cluster (ssh into the head node) with ``ray attach config.yaml``. For a full reference on the Ray CLI, please refer to :ref:`the cluster commands reference <cluster-commands>`.
+
+To finish, don't forget to shut down the cluster. Run the following command:
```

Comment:
Add something about auto shutdown / idle minutes here so users know this is an option

Reply:
idle minutes only applies to worker nodes, the head node never kills itself, so you should always run `ray down`

Reply:
this is for OSS, right?